conf/layer.conf

```diff
@@ -29,6 +29,9 @@ AOS_MAIN_NODE_HOSTNAME ?= "main"
 
 AOS_UM_COMPONENT_PREFIX ?= ""
 
+AOS_IDENTIFIER_PLUGIN_NAME ?= "visidentifier"
+AOS_IDENTIFIER_PLUGIN_PARAMS ?= "VISServer=wss://wwwivi:443"
```

Comment:
Just a suggestion: by default, WSS uses port 443, so there's no need to explicitly specify port 443.